terraform/environments/digital-prison-reporting/operational_datastore.tf

```diff
@@ -47,4 +50,20 @@ resource "aws_security_group" "glue_operational_datastore_connection_sg" {
     cidr_blocks = ["0.0.0.0/0"]
     description = "Allow all traffic out from this Security Group"
   }
-}
\ No newline at end of file
+}
+
+# This SG should be attached to the Operational DataStore to allow the transfer component lambda to run migrations
```

Comment:
For the time being this is manually attached to the temporary Operational DataStore which is not managed by terraform.